src/uncommenter.rs

```diff
@@ -16,6 +16,11 @@ pub fn uncomment(text: &str) -> String {
     for c in text.chars() {
         match (state, c) {
             (State::Default, '/') => state = State::StartComment,
+            // We now also treat preprocessor directives like they are comments.
```

Comment:
Thanks for working on this. This seems okay to me. Please just add a comment here that this is a hack to get more IPDL files with preprocessor things working, so nobody thinks this is a full fix.